RELEASE-NOTES.md

```diff
@@ -104,24 +104,97 @@ buckets.default.allow_mult = true
+written in `riak_kv` section of `advanced.config`. See
 [Setting up the Proper Riak Backend](http://docs.basho.com/riakcs/1.5.4/cookbooks/configuration/Configuring-Riak/#Setting-up-the-Proper-Riak-Backend)
-for details.
+for details. `app.config` should be removed when `advanced.config`
+should be used.
+
+#### Redesign of memory sizing
+
+It is also worth noting that since LevelDB has changed its
+configuration item on memory size configuration, from `max_open_files`
+to `total_leveldb_mem_percent`. This specifies total amount of memory
+consumed by LevelDB. This configuration is very important because the
+default value of memory cap limitation has changed from proportional
+number of `max_open_files` to specifying percentage of system's
+physical memory size.
+
+Configuring `total_leveldb_mem_percent` is *strongly recommended* as
+its default value is
+[70%](https://github.com/basho/eleveldb/blob/2.0.2/priv/eleveldb.schema#L17),
+which might be too aggressive for multi-backend configuration that
+also uses bitcask. Bitcask keeps its keydir on memory, which could be
+fairly large depending on use cases.
+
+Hereby basic idea of redisigning memory sizing is described: estimate
+total number of objects with their size distribution. Total number of
+keys `N(b)` in Bitcask accross the whole cluster will be:
+
+```
+N(b) = N(o, size <= 1MB) + avg(o, size > 1MB)
+```
+
+where `N(o, size <= 1MB)` is number of objects whose size is less than
+1MB, and `avg(o, size > 1MB)` is average size of objects whose size is
+more than 1MB. If average lifetime of objects is comparably smaller
+than leeway period, take objects waiting for garbage collections into
+account as live objects on disk.
+
+Once numbers of keys figured out, estimate the amount of memory used
+by Bitcask keydir with
+[Bitcask Capacity Planning](http://docs.basho.com/riak/2.0.5/ops/building/planning/bitcask/).
 
-For later release than Riak 2.0.5, new configuration in Riak will be
-just
+Bucket size is always 19B (see `riak_cs_utils:to_bucket_name/2`), Key
+size is always 20B (see `riak_cs_lfs_utils:block_name/3`). Average
+Value Size might be 1MB if large objects are dominant, otherwise it
+should be estimated under specific usecase. Number of writes is 3.
+
+Note that this only takes bitcask into account, so another room of
+memory for LevelDB should be preserved as described above. Moreover,
+to improve IO performance, extra space for kernel disk cache would
+desirable:
+
+```
+Memory for storage = (Memory for backends) + (Memory for kernel cache)
+Memory for backends = (Memory for Bitcask) + (Memory for LevelDB)
+```
+
+Note: `leveldb.maximum_memory_percent` (or `leveldb.maximum_memory`)
+in `riak.conf` also can be used. On the other hand, there are a way to
+use cuttlefish-style items starting with `multi_backend.be_default...`
+could be used in `riak.conf`, but that is confusing and less simple
+than recommended way described above.
```

Comment:
I'm wondering whether this sentence is necessary because it seems advanced topic. Riak CS uses only one kind of leveldb, then `total_leveldb_mem_percent` is sufficient, isn't it?